spec/unit/hanami/router/redirect_spec.rb

```diff
@@ -12,8 +12,14 @@
       env = Rack::MockRequest.env_for("/redirect")
       status, headers, = router.call(env)
 
+      location_header = if Hanami::Router.modern_rack?
+                          headers.fetch("location")
```

Comment:
```suggestion
                          headers["location"]
```

Having the `.fetch` on one side of the conditional and not the other imparts special emphasis on the fetch, which I don't think is warranted here. Even if `headers` don't have a `"location"` key (which would fail the fetch), if we used `headers["location"]` instead, we'd get a `nil` and still end up with a failing expectation a few lines below, which is a more useful outcome from the test than a `KeyError`.

(This suggestion applies to here as well as the few other places in our specs where we're doing a similar thing)

Reply:
While we're here, let's use sensible indentation 😅 

```ruby
headers =
  if blah?
    "blorp"
  else
    "fneep"
  end
```

I'm happy with basically anything except the indentation style you've used here 😆

Reply:
_lol_ point taken! i'll give change it to my second favorite indentation style 😉